crates/sui-e2e-tests/tests/bridge_tests.rs

```diff
@@ -63,102 +53,8 @@ async fn test_committee_registration() {
     let test_cluster: test_cluster::TestCluster = TestClusterBuilder::new()
         .with_protocol_version(37.into())
         .with_epoch_duration_ms(10000)
-        .build()
+        .build_with_bridge()
```

Comment:
nit: is it better to have `with_bridge` then `build`, instead of a new build method for bridge?